src/syncer.rs

```diff
@@ -229,6 +229,54 @@ pub enum Task {
     BroadcastTransaction(BroadcastTransaction),
 }
 
+impl fmt::Display for Task {
```

Comment:
If you want to implement a deeper `Display` you can use the derive macro (we already depend on it) like here: https://github.com/farcaster-project/farcaster-core/blob/main/src/bundle.rs#L534-L535. It is part of `amplify` or something similar and there is nice derivation for Enums.

Reply:
Changed it to use the macro.